HW6/scripts/Homework_6.sql

```diff
@@ -0,0 +1,28 @@
+SHOW ERRORS;
```

Comment:
JUST SUGGESTION AND THIS COMMENT IS NOT RELATED TO ASSIGNMENT6.

Since this assignment is not specific to validating the rolling average of each batter, I am not looking in the sql file changes. Otherwise "ROWS BETWEEN 99 PRECEDING AND CURRENT ROW" is a bug which we discussed in assignment 2.

Reply:
Yes.I changed sql code to get actual rolling average according to days and corrected this error of PARTITION by rows. Thanks for the comment.